spec/lib/traject/macros/custom_spec.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+RSpec::Matchers.define_negated_matcher :not_include, :include
+
+RSpec.describe 'Macros spec:' do
+  let(:leader) { '1234567890' }
+
+  let(:field) { 'full_links_struct' }
+  let(:url_856) do
+    { '856' => { 'ind1' => '0', 'ind2' => '0', 'subfields' => [{ 'u' => 'https://scholarsphere.psu.edu/files/02870v85d' },
+                                                               { '3' => 'How to use the collection' }] } }
+  end
+  let(:result) { @indexer.map_record(MARC::Record.new_from_hash('fields' => [url_856], 'leader' => leader)) }
+
+  before(:all) do
+    c = './lib/traject/psulib_config.rb'
+    @indexer = Traject::Indexer.new
+    @indexer.load_config_file(c)
+  end
+
+  describe 'A record with a fulltext link because of indicator 2 is 0' do
```

Comment:
@cdmo this is minor: the description says "indicator 2 is 0" but the mocked field seems to have ind2 as 1 🤔

Reply:
🤦‍♂️